### PR TITLE
RUBY-167, RUBY-168, RUBY-169, RUBY-170, RUBY-179, RUBY-180, RUBY-181

### PR DESCRIPTION
**Pull request by stamhankar999**

### CODE REVIEW

File: lib/cassandra/column_container.rb [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+    end
+
+    # @private
+    def inspect
+      "#<#{self.class.name}:0x#{object_id.to_s(16)} " \
+          "@keyspace=#{@keyspace} @name=#{@name}>"
+    end
+
+    # @private
+    def eql?(other)
```
Comment (avalanche123) [inline lib/cassandra/column_container.rb]: this should be unnecessary as both keyspace and materialized view implement their own

Comment (avalanche123) [inline lib/cassandra/column_container.rb]: `ColumnContainer` is an equivalent of abstract class, so I don't see how we would ever compare two

**Comment by avalanche123** [comment]
looks like some unit tests are broken

File: lib/cassandra/column_container.rb (hunk 1 above)
Comment (stamhankar999) [inline lib/cassandra/column_container.rb]: Good call on inspect; will remove. eql? is invoked by the subclasses' eql?'s to compare the ColumnContainer-owned fields.

**Comment by stamhankar999** [comment]
Yikes; will fix that.

**Pull request merged**